v1.1/create-security-certificates-openssl.md

```diff
@@ -0,0 +1,329 @@
+
+<div id="toc"></div>
+
+## Subcommands
+
+Subcommand | Usage
+-----------|------
+[`openssl genrsa`](https://wiki.openssl.org/index.php/Manual:Genrsa(1)) | Create an RSA private key.
+[`openssl req`](https://wiki.openssl.org/index.php/Manual:Req(1)) | Create CA certificate and CSRs (certificate signing requests).
+[`openssl ca`](https://wiki.openssl.org/index.php/Manual:Ca(1)) | Create node and client certificates using the CSRs.
+
+## Configuration Files
+
+To use [`openssl req`](https://wiki.openssl.org/index.php/Manual:Req(1)) and [`openssl ca`](https://wiki.openssl.org/index.php/Manual:Ca(1)) subcommands, you need the following configuration files:
+
+File name pattern | File usage
+-------------|------------
+`ca.cnf`     | CA configuration file
+`node.cnf`   | Server configuration file
+`client.cnf` | Client configuration file
+
+## Certificate Directory
+
+To create node and client certificates using the OpenSSL commands, you need access to a local copy of the CA certificate and key. We recommend creating all certificates (node, client, and CA certificates), and node and client keys in one place and then distributing them appropriately. Store the CA key somewhere safe and keep a backup; if you lose it, you will not be able to add new nodes or clients to your cluster. 
+
+Use the [`openssl genrsa`](https://wiki.openssl.org/index.php/Manual:Genrsa(1)) and [`openssl req`](https://wiki.openssl.org/index.php/Manual:Req(1)) subcommands to create all certificates, and node and client keys in a single directory, with the files named as follows:
+
+File name pattern | File usage
+-------------|------------
+`ca.crt`     | CA certificate
+`node.crt`   | Server certificate
+`node.key`   | Key for server certificate
+`client.<user>.crt` | Client certificate for `<user>` (for example: `client.root.crt` for user `root`)
+`client.<user>.key` | Key for the client certificate
+
+Note the following:
+
+- The CA key should not be uploaded to the nodes and clients, so it should be created in a separate directory.
+
+- Keys (files ending in `.key`) must not have group or world permissions (maximum permissions are 0700, or `rwx------`). This check can be disabled by setting the environment variable `COCKROACH_SKIP_KEY_PERMISSION_CHECK=true`.
+
+## Examples
+
+### Create the CA key and certificate pair
+
+1. Create two directories:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ mkdir certs
+    ~~~
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ mkdir my-safe-directory
+    ~~~
+    - `certs`: Create your CA certificate and all node and client certificates and keys in this directory and then upload the relevant files to the nodes and clients.
+    - `my-safe-directory`: Create your CA key in this directory and then reference the key when generating node and client certificates. After that, keep the key safe and secret; do not upload it to your nodes or clients.
+
+2. Create the `ca.cnf` file:
```

Comment:
nit: I think we can simplify this to

> 2. Create the `ca.cnf` file and copy the following configuration into it.

with that sentence removed from the paragraph following. So that paragraph would just start with `You can set the...`.